src/components/LoadingCard.tsx

```diff
@@ -2,23 +2,25 @@ import {
   charityIdFromItem,
   fetchItemMetadata,
   mediaUriFromItem,
+  priceFromItem,
+  StoreThing,
 } from '../state/items'
 import Card, { CardShimmer } from '../components/Card'
 import { useRecoilValueLoadable } from 'recoil'
 import React from 'react'
 
 const LoadingCard: React.FC<{
-  id: string
-}> = ({ id }) => {
-  const metadata = useRecoilValueLoadable(fetchItemMetadata({ id }))
+  thing: string | StoreThing
+}> = ({ thing }) => {
+  const metadata = useRecoilValueLoadable(fetchItemMetadata({ thing }))
 
   if (metadata.state === 'hasValue' && metadata.contents) {
     return (
       <Card
-        id={id}
-        username={metadata.contents.minter}
-        title={metadata.contents.thing.title}
-        price={{ fraction: 1, token: 'NEAR' }}
+        id={metadata.contents.thing.id}
```

Comment:
Let's pass a single object to Card with all properties

Reply:
I plan to do it in a separate PR, to keep history clean